File: src/main/java/Arrays/Actv1.java
//DECLARACION Y CREACION DE UN ARREGLO
//DELCARAR UN ARREGLO DE 10 ENTEROS
//INICIALIZARLO CON VALORES ALEATORIOS

package Arrays;
import java.util.Arrays;

public class Actv1 {

    public int[] arreglo = new int[10];

    public int[] llenarArreglo(){
        for (int i = 0; i < arreglo.length; i++) {
            arreglo[i] = (int)(Math.random()*100);
        }
        System.out.println("Arreglo creado: " + Arrays.toString(arreglo));
        return arreglo;
    }

}

File: ArraysACTV/Actv1.cs
/*
Declaración y creación de un arreglo
Declarar un arreglo de 10 enteros en cada lenguaje dado
Inicializarlo con valores aleatorios.
*/

class Actv1
{
    public static int[] CrearArreglo(){
        // Declaración y creación de un arreglo de 10 enteros
        int[] arreglo = new int[10];

        // Inicialización con valores aleatorios
        Random rand = new Random();
        for (int i = 0; i < arreglo.Length; i++)
        {
            arreglo[i] = rand.Next(1, 101); // Genera un número aleatorio entre 1 y 100
        }

        // Imprimir el arreglo para verificar los valores
        Console.WriteLine("Arreglo de enteros con valores aleatorios:");
        foreach (int numero in arreglo)
        {
            Console.WriteLine(numero);
        }

        return arreglo;
    }
}